templates/desktop/flavours.html

```diff
@@ -4,72 +4,80 @@
 {% block meta_copydoc %}https://docs.google.com/document/d/1M5nkigcSRzq1YVjiXOZkeU4EUCqT0uYBR3Y_sjxM-wM/edit{% endblock meta_copydoc %}
 
+{% block body_class %}is-paper{% endblock %}
 
 {% block content %}
 
-<section class="p-strip--suru-topped">
-  <div class="row">
-    <div class="col-6">
-      <h1>Ubuntu flavours</h1>
-      <p class="p-heading--3">What are Ubuntu flavours?</p>
-      <p>Ubuntu flavours offer a unique way to experience Ubuntu, each with their own choice of default applications and settings. Ubuntu flavours are owned and developed by members of our global community and backed by the full Ubuntu archive for packages and updates.</p>
-    </div>
-    <div class="col-6 u-align--center u-vertically-center u-hide--small u-hide--medium">
-      {{ image (
-        url="https://assets.ubuntu.com/v1/e84a9007-ubuntu-flavours-23.png",
-        alt="",
-        width="1296",
-        height="460",
-        hi_def=True,
-        loading="auto"
-        ) | safe
-      }}
+<section class="p-strip--whitesuru is-shallow u-no-padding--bottom">
+  <div class="p-section--deep">
+    <div class="row">
+      <div class="col-start-large-4 col-9">
+        <div class="row">
+          <div class="col-medium-3">
+            <h1>Ubuntu flavours</h1>
+          </div>
+          <div class="col-medium-3">
+            <div class="p-section--shallow">
+              <p class="p-heading--2">What are Ubuntu flavours?</p>
+              <p>Ubuntu flavours offer a unique way to experience Ubuntu, each with their own choice of default applications and settings. Ubuntu flavours are owned and developed by members of our global community and backed by the full Ubuntu archive for packages and updates.</p>
+            </div>
+            {{
+              image (
+                url="https://assets.ubuntu.com/v1/8527100a-ubuntu-flavours-23.png",
+                alt="",
+                width="518",
+                height="173",
+                hi_def=True,
+                loading="auto",
+                ) | safe
+              }}
+            </div>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-start-large-3 col-6">
+
+      </div>
     </div>
   </div>
 </section>
 
-<section class="p-strip--light">
-  <div class="row">
-    <div class="col-6">
+<section class="p-section">
+  <div class="row--50-50">
+    <hr class="p-rule" />
+    <div class="col">
       <div class="p-heading-icon">
         <div class="p-heading-icon__header">
           {{
             image(
               url="https://assets.ubuntu.com/v1/a2f090ef-edubuntu-logo.svg",
-              alt="Edubuntu icon",
+              alt="",
               height="64",
               width="64",
               hi_def=True,
               loading="lazy",
-              attrs={"class": "p-heading-icon__img"}
+              attrs={"class": "p-heading-icon__img", "style": "margin-top: 0.5rem;"},
```

Comment:
You could use [this style](https://github.com/canonical/ubuntu.com/blob/main/static/sass/styles.scss#L1251) in place of inline styles here